doc/releases/pacific.rst

```diff
@@ -5,6 +5,182 @@ Pacific
 Pacific is the 16th stable release of Ceph.  It is named after the
 giant pacific octopus (Enteroctopus dofleini).
 
+v16.2.5 Pacific
+===============
+
+This is the fifth  backport release in the Pacific series.  We recommend all users
+update to this release.
+
+Notable Changes
+--------------
+* .github/labeler: add api-change label (`pr#41818 <https://github.com/ceph/ceph/pull/41818>`_, Ernesto Pu
+erta)
```

Comment:
This and many other lines are still wrapped.  It should be one physical line per change like in the second half of the changelog -- all wrapped lines seem to be in the first half for some reason.